.github/workflows/ci.yml

```
@@ -124,7 +124,7 @@ jobs:
     needs: test-modules
     strategy:
       matrix:
-        module: [cockroachdb, consul, nginx, toxiproxy]
+        module: [cockroachdb, consul, minio, nginx, toxiproxy]
```

Not needed, as it's already present in the modules group
```suggestion
        module: [cockroachdb, consul, nginx, toxiproxy]
```